config-schema.yml

```diff
@@ -1,10 +1,14 @@
 type: object
-additionalProperties: false
+additionalProperties: true
```

Comment:
It seems like we should set `additionalProperties` to `true` everywhere. 

WDYT?